ui/packages/lib/src/components/nav_drawer/NavDrawer.js

```diff
@@ -52,29 +39,123 @@ export const NavDrawer = ({ homeUrl = "/", appLinks }) => {
+    },
+    {
+      label: "Turing User Guide",
+      href:
+        "https://go-jek.atlassian.net/wiki/spaces/DSP/pages/1757883181/Turing+User+Documentation"
+    },
+    {
+      label: "Feast User Guide",
+      hrefh: "https://docs.feast.dev/user-guide/overview"
+    }
+  ];
+
+  const [navIsOpen, setNavIsOpen] = useState(true);
+  const [navIsDocked, setNavIsDocked] = useState(true);
+
+  useEffect(() => {
+    setNavIsDocked(navIsOpen);
+  }, [navIsOpen]);
+
+  const [appExpanded, toggleAppExpanded] = useToggle(true);
+  const [docsExpanded, toggleDocsExpanded] = useToggle(true);
   return (
-    <EuiNavDrawer>
+    <EuiCollapsibleNav
+      aria-label="Main navigation"
+      isOpen={navIsOpen}
+      showCloseButton={false}
+      isDocked={navIsDocked}
+      showButtonIfDocked={true}
+      style={{ top: "49px", height: "calc(100% - 49px)", width: "262px" }}
+      button={
+        <EuiHeaderSectionItemButton
+          aria-label="Toggle main navigation"
+          onClick={() => setNavIsOpen(!navIsOpen)}>
+          <EuiIcon type={"menu"} size="m" aria-hidden="true" />
+        </EuiHeaderSectionItemButton>
+      }
+      onClose={() => setNavIsOpen(false)}>
       <EuiFlexGroup
         gutterSize="none"
         direction="column"
         style={{ height: "100%" }}>
-        <EuiFlexItem grow={true} style={{ marginTop: `${topPosition}px` }}>
+        <EuiFlexItem grow={true}>
           {appLinks && (
             <>
-              <EuiNavDrawerGroup showToolTips={true} listItems={appLinks} />
+              <EuiCollapsibleNavGroup listItems={appLinks} />
               <EuiHorizontalRule margin="none" />
             </>
           )}
-          <EuiNavDrawerGroup showToolTips={true} listItems={mlpLinks} />
-          <EuiHorizontalRule margin="none" />
+          <EuiCollapsibleNavGroup
+            isCollapsible={true}
+            initialIsOpen={appExpanded}
+            onToggle={toggleAppExpanded}
+            iconType="apps"
+            title="Products">
+            <EuiListGroup
+              aria-label="MLP apps"
+              listItems={mlpLinks}
+              maxWidth="none"
+              color="text"
+              gutterSize="s"
+              size="s"
+            />
+          </EuiCollapsibleNavGroup>
+
+          <EuiCollapsibleNavGroup
+            title="Learn"
+            iconType="training"
+            isCollapsible={true}
+            initialIsOpen={docsExpanded}
+            onToggle={toggleDocsExpanded}>
+            <EuiListGroup
+              aria-label="Learn" // A11y : EuiCollapsibleNavGroup can't correctly pass the `title` as the `aria-label` to the right HTML element, so it must be added manually
+              listItems={docLinks}
+              maxWidth="none"
+              color="subdued"
+              gutterSize="s"
+              size="s"
+            />
+          </EuiCollapsibleNavGroup>
         </EuiFlexItem>
+
+        <EuiHorizontalRule margin="none" />
```

Comment:
Please move this rule after the Project Setting button. Similar to the previous sidebar:
![image](https://user-images.githubusercontent.com/8122852/113117685-e7389100-9238-11eb-859a-7919d94bbcf2.png)